<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>340</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="errBrowser">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>320</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>31</width>
      <height>411</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="StudentsLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Студент</string>
    </property>
   </widget>
   <widget class="QLabel" name="SubjectsLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Предмет</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="firstList">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>521</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="StudentsWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="SubjectsWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SubjectsNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SubjectsNameEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="SubjectsHourEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>1071</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>300</y>
      <width>31</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="SubjectsHoursLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во часов</string>
    </property>
   </widget>
   <widget class="QLabel" name="SubjectsTeacherName">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя преподавателя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SubjectsTeacherNameEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавление элементов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить связь</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Убрать связь</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConPortEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConFlightEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ConPortLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Предмет</string>
    </property>
   </widget>
   <widget class="QLabel" name="ConFlightLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Студент</string>
    </property>
   </widget>
   <widget class="QLabel" name="SubjectsSemesterLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер семестра</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SubjectsSemesterEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Предмет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Студент</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="StudenstNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО студента</string>
    </property>
   </widget>
   <widget class="QLabel" name="StudentsPolLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Пол</string>
    </property>
   </widget>
   <widget class="QLabel" name="StudentsBornLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата рождения</string>
    </property>
   </widget>
   <widget class="QLabel" name="StudentsNumberLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>№ Зачетки</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="StudentsNameEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="StudentsPolEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="StudentsBornEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="StudentsNumberEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="OpenAction"/>
   <addaction name="separator"/>
   <addaction name="actionStOpenAction"/>
   <addaction name="save"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="OpenAction">
   <property name="text">
    <string>Открыть файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionStOpenAction">
   <property name="text">
    <string>Открыть стандартный файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Сохранить в файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
